core/state/statedb.go

```diff
@@ -1060,12 +1076,21 @@ func (s *StateDB) PrepareAccessList(sender common.Address, dst *common.Address,
 	for _, addr := range precompiles {
 		s.AddAddressToAccessList(addr)
 	}
+
+	// Note: If an address is specified multiple times in the access list, only the
+	// last storage slots provided for it are used in predicates.
```

Comment:
Calling this out because this is a very weird invariant that could be error prone moving forward.